Extension/lib/content-script/websocket.js

```diff
@@ -0,0 +1,296 @@
+/**
+ * This file is part of Adguard Browser Extension (https://github.com/AdguardTeam/AdguardBrowserExtension).
```

Comment:
! Add a link to the uBlock WebSocket wrapper, you've reused a lot of code from there.